src/components/Table.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+
+
+class Table extends React.Component {
+    constructor( props ) {
+        super( props )
+        this.state = {
+            translationData: []
+        };
+    }
+
+    renderTableData() {
+        if ( this.state.translationData.length !== 0 ) {
+            return this.state.translationData.map( ( translation, index ) => {
+
+                const [ chinese, pinyin, english, italian, arabic, serbian, croatian, russian, german, hebrew, french,
+                    hungarian, slovak, spanish, portugues, turkce, greek, romanian ]
+                    = translation // Destructuring.
+
+                return (
+                    <tr key={ index }>
+                        <td>{ chinese + " " + pinyin }</td>
+                        <td>{ english }</td>
+                        <td>{ italian }</td>
+                        <td>{ arabic }</td>
+                        <td>{ serbian }</td>
+                        <td>{ croatian }</td>
+                        <td>{ russian }</td>
+                        <td>{ german }</td>
+                        <td>{ hebrew }</td>
+                        <td>{ french }</td>
+                        <td>{ hungarian }</td>
+                        <td>{ slovak }</td>
+                        <td>{ spanish }</td>
+                        <td>{ portugues }</td>
+                        <td>{ turkce }</td>
+                        <td>{ greek }</td>
+                        <td>{ romanian }</td>
+                    </tr>
+                )
+            })
+        } else {
+            return (
+                <tr>
+                    <td>{"loading"}</td>
+                </tr>
+            )
+        }
+    }
+
+    componentDidMount() {
+        fetch('https://cmlgbackend.wdcc.co.nz/translations')
+            .then( results => {
+                return results.json();
+            })
+            .then( data => {
+                let sortedArray = [];
+                let innerArray = [];
+                let dataIndex;
+                let currentData;
+
+                for ( dataIndex = 0; dataIndex < data.length; dataIndex++ ) {
+                    currentData = data[ dataIndex ];
+
+                    // innerArray.length starts from 0 whereas language_id starts from 1.
+                    // Check if the translated word is in the correct column (under the correct language). So if the
+                    // English translation for the word is not under English, throw an exception.
+                    if ( innerArray.length + 1 === currentData.language_id ) {
+                        innerArray[ innerArray.length ] = currentData.name;
+                    } else {
+                        throw new Error( "The translated word does not match the language." );
+                    }
+
+                    // Number of languages
+                    const innerArrayLength = 18;
```

Comment:
Is there a better name than innerArrayLength? Because 18 means the total number of languages, it might be better to consider giving it a name like numberOfLanguages.